src/components/KanaQuiz/KanaQuiz.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { KanaType } from '../../types/kana';
+import KanaQuizForm from './KanaQuizForm';
+import { QuizStats } from '../../types/kanaQuiz';
+
+type KanaQuizProps = {
+  kana: KanaType[];
+};
+
+export default function KanaQuiz({ kana }: KanaQuizProps) {
```

Comment:
You don't need to create a new type here if it's only for this function. You can also do: `export default function KanaQuiz({ kana }: { kana: KanaType[] }) {}`